beetsplug/mbsubmit.py

```diff
@@ -56,7 +60,44 @@ def __init__(self):
 
     def before_choose_candidate_event(self, session, task):
         if task.rec <= self.threshold:
-            return [PromptChoice("p", "Print tracks", self.print_tracks)]
+            return [
+                PromptChoice("p", "Print tracks", self.print_tracks),
+                PromptChoice(
+                    "f",
+                    'print tracks into a "beets-unmatched" File',
+                    self.markunmatched,
+                ),
+                PromptChoice(
+                    "o", "Open Picard with the relevent files", self.picard
+                ),
+            ]
+
+    def markunmatched(self, session, task):
+        fmt = self.config["format"].as_str()
+        for item in task.items:
+            unmatched_path = (
+                Path(item.path.decode("utf-8")).parent / "beets-unmatched"
+            )
+            with open(unmatched_path, "a") as p:
+                p.write(format(item, fmt) + "\n")
+
+    def picard(self, session, task):
+        picard_path = which(self.config["picard-path"].as_str())
```

Comment:
You could use the .as_pathname method from confuse here, which gives you a basic sanity check. 

An entirely different approach further down below could be to use a try except wrapper around the subprocess call. Then catch a file not found error gracefully.

Reply:
> You could use the .as_filename method from confuse here, which gives you a basic sanity check.

Good idea :) Thanks.